frontend/sozisel-app/src/components/ParticipantActiveSession/ParticipantActiveSession.tsx

```diff
@@ -74,6 +75,7 @@ export default function ParticipantActiveSession({
     if (sessionNotifications?.sessionNotifications) {
       switch (sessionNotifications.sessionNotifications.info) {
         case SessionInfo.SessionEnd:
+          localStorage.removeItem(PARTICIPANT_TOKEN);
```

Comment:
This solution is partially wrong. If somebody leaves the session by himself and the session ends afterward then he can rejoin it as well. You have to check the `sessionEndTime` or whatever is available via session query and there decide if you can enter the session or redirect.

Reply:
it's not available via `SessionThumbnails` query, so I made such workaround.